### PR TITLE
New structure for model files (clean up) and new power law function

### PR DESCRIPTION
**Pull request by grburgess**
I have started cleaning up the functions file as it is getting crowded. I moved power law functions
to their own file as well as added a new power law normalized to its energy flux.

There are additionally some improved import statements to keep from polluting the environment.

### CODE REVIEW

File: astromodels/functions/dark_matter/dm_models.py [hunk 1]
```diff
@@ -288,8 +288,10 @@ def _setup(self):
                                                   fill_value=None)
 
         if self.channel.value in [1, 6, 7] and self.mass.value > 10000.:
-            print("ERROR: currently spectra for selected channel and mass not implemented.")
-            print("Spectra for channels ['ee','gg','WW'] currently not available for mass > 10 TeV")
+            print(
```
Comment (henrikef) [inline astromodels/functions/dark_matter/dm_models.py:291]: Change print to logging?

**Review by henrikef — CHANGES_REQUESTED** [review]
See individual comments (just one small issue, really).

I'm getting tons of `NameError: name 'np' is not defined` in threeML with this version of astromodels. I assume this is related to changes in importing numpy. I'll try and see if I can add `import numpy as np` to threeML where needed. We should wait for that to be ready though before merging this.

**Comment by grburgess** [comment]
> See individual comments (just one small issue, really).
> 
> I'm getting tons of `NameError: name 'np' is not defined` in threeML with this version of astromodels. I assume this is related to changes in importing numpy. Fixed in [threeML/threeML#448](https://github.com/threeML/threeML/pull/448)

Ja, I cleaned this up because we were just importing everything from the modules with the import * stuff... in principle it is fine, but it might tick off someone wanting to use certain parts of 3ML etc. Also that was some baking coding on our part because we should've been importing those modules directly :)

File: astromodels/functions/dark_matter/dm_models.py (hunk 1 above)
Comment (grburgess) [reply astromodels/functions/dark_matter/dm_models.py:291]: could do!

File: astromodels/functions/__init__.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+
+
+           ]
+
+if has_atomdb:
+    __all__.extend(["APEC", "VAPEC"])
+
+
+if has_naima:
```
[the remark this answers is not in the document]
Comment (grburgess) [reply astromodels/functions/__init__.py]: woops. yup. fixed

**Review by henrikef — APPROVED** [review]
Looks good now, although I recommend waiting for https://github.com/threeML/threeML/pull/448 (CI keeps crashing during the tests due to low disk space, but everything worked locally)

**Comment by grburgess** [comment]
@henrikef thanks! We can discuss in a few moments.

**Pull request merged**